src/actions/profile-view.js

```diff
@@ -591,34 +591,12 @@ export function hideGlobalTrack(trackIndex: TrackIndex): ThunkAction<void> {
 
     // Find another selectedThreadIndex if the current selected thread is hidden
     // with this operation.
-    if (globalTrackToHide.type === 'process') {
```

Comment:
I needed to use this part of the thunk action. Extracted into another function so I can reuse that. I wanted it to be similar with what we do for `_findOtherVisibleThread`.